airbyte-commons/src/main/java/io/airbyte/commons/enums/Enums.java

```diff
@@ -32,6 +32,10 @@
 public class Enums {
 
   public static <T1 extends Enum<T1>, T2 extends Enum<T2>> T2 convertTo(T1 ie, Class<T2> oe) {
```

Comment:
not failing on null seems desirable here. if i'm missing something though, i'm happy to create split this into 2 methods. one that accepts nulls and one that does not.

Reply:
fair